packages/server/src/index.js

```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const { CronJob } = require('cron');
 const fs = require('fs').promises;
 const path = require('path');
+require('dotenv').config();
```

Comment:
dotenv is called in ./configure.js in line 7 so I think we can remove it. Did you have a problem with env vars?